docfx.json

```
@@ -162,12 +162,34 @@
         },
         "fileMetadata": {
             "feedback_system": {
+                "docs/**.*": "OpenSource",
                 "docs/standard/design-guidelines/**/**.md": "None",
                 "docs/framework/data/adonet/**/**.md": "None",
                 "docs/framework/data/wcf/**/**.md": "None",
                 "docs/framework/ui-automation/**/**.md": "None",
                 "docs/framework/wcf/**/**.md": "None"
             },
+            "open_source_feedback_contributorGuideUrl": {
+                "docs/**/**.*": "https://learn.microsoft.com/contribute/content/dotnet/dotnet-contribute"
+            },
+            "open_source_feedback_issueUrl": {
+                "docs/**.*": "https://github.com/dotnet/docs/issues/new?template=customer-feedback.yml"
+            },
+            "open_source_feedback_productName": {
+                "docs/**.*": ".NET feedback"
+            },
+            "open_source_feedback_productDescription": {
+                "docs/**.*": "The .NET documentation is open source. Provide feedback here."
+            },
+            "open_source_feedback_issueTitle": {
+                "docs/**.*": "[Customer issue] - "
```

Let's leave the title blank:

```suggestion
                "docs/**.*": ""
```